client/sources/doctest/__init__.py

```diff
@@ -0,0 +1,32 @@
+from client.sources.common import importing
+from client.sources.doctest import models
+import logging
+import os
+
+log = logging.getLogger(__name__)
+
+def load(file, name, args):
+    """Loads doctests from a specified filepath.
+
+    PARAMETERS:
+    file -- str; a filepath to a Python module containing OK-style
+            tests.
+
+    RETURNS:
+    Test
+    """
+    if not os.path.isfile(file) or not file.endswith('.py'):
+        log.info('Cannot import doctests from {}'.format(file))
+        # TODO(albert): raise appropriate error
+        raise Exception
```

Comment:
At the bare minimum, you should have a message here so that we can appropriately diagnose it when students run into issues.